aries-site/src/examples/templates/wizard/WizardValidationExample.js

```diff
@@ -174,48 +144,35 @@ const data = [
 
 const StepThree = () => {
   return (
-    <>
-      <Box gap="small" margin={{ bottom: 'large' }}>
-        <List data={data} pad={{ horizontal: 'none', vertical: 'small' }}>
-          {(datum, index) => (
-            <Box key={index} direction="row" gap="medium" align="center">
-              <Checkmark color="text-strong" size="small" />
-              <Text color="text-strong">{datum}</Text>
-            </Box>
-          )}
-        </List>
-        <Text color="text-strong">
-          Include guidance to what will occur when “Finish Setup” is clicked.
-        </Text>
-      </Box>
-      <Button fill="horizontal" label="Finish Setup" primary type="submit" />
-    </>
+    <Box gap="small" margin={{ bottom: 'medium' }}>
+      <List data={data} pad={{ horizontal: 'none', vertical: 'small' }}>
+        {(datum, index) => (
+          <Box key={index} direction="row" gap="medium" align="center">
+            <Checkmark color="text-strong" size="small" />
+            <Text color="text-strong">{datum}</Text>
```

Comment:
I believe the Figma file has this as a medium (500) weight, but the file is not referencing the componentized font treatments. I think this is causing translation issues between the design intent (weight and color differentiation between the paragraph text and the check list) and what we are actually coding.

THink it would be worthwhile to have @vavalos5 to take a look and see if this file needs some "kitchen cleaning" so that we can maintain alignment here.

Reply:
I've opened an issue to track the design discussions: #1302